src/nuxt.ts

```diff
@@ -9,16 +9,32 @@ export async function loadNuxt () {
   ctx.nuxt = new Nuxt(ctx.options.config)
 }
 
-export async function loadFixture () {
+const resolveRootDir = () => {
   const { options } = getContext()
+  const dirs = [options.rootDir, resolve(options.testDir, options.fixture), process.cwd()]
+
+  const isNuxtApp = (dir: string) => {
```

Comment:
It would be nice movie it to a top level function

Reply:
I thought about it, and also about exporting it

Reply:
@pi0 I don't know if it makes a lot of sense to export them, does it?

Reply:
Thanks. As test-utils probably not but we may add it to nuxt/kit (i will invite you to repo soon btw)